notification/shoutrrr.go

```diff
@@ -88,11 +89,32 @@ func Send(ctx *api.Context, connectionName, shoutrrrURL, title, message string,
 
 	injectTitleIntoProperties(service, title, allProps)
 
-	var params *types.Params
+	params := &types.Params{}
 	if properties != nil {
 		params = (*types.Params)(&allProps)
 	}
 
+	// NOTE: Until shoutrrr fixes the "UseHTML" props, we'll use the mailer package
+	if service == "smtp" {
```

Comment:
But now this doesn't work for smtp://system ?

Reply:
It does. At this point in code, the url is never `smtp://system`. It's already prepopulated. https://github.com/flanksource/incident-commander/blob/cf47ca1327b40638ce8e569691c270d3d6364723/notification/shoutrrr.go#L54-L60